### PR TITLE
display contact details tab in view organisation details screen

### PR DESCRIPTION
**Pull request by stevelee-civica**
display fields in contact details  tab

### CODE REVIEW

File: src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml [hunk 1]
```diff
@@ -18,12 +18,68 @@
             <h1 class="govuk-heading-m">
                 @title
             </h1>
-            <table id="pcs-details-table" class="govuk-table">
+            <table id="pcs-contact-details-table" class="govuk-table">
                 <tbody class="govuk-table__body">
-                   
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">First Name</th>
+                        <td class="govuk-table__cell">@Model.SmallProducerSubmissionData.CurrentSubmission.ContactData.FirstName</td>
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml:25]: @stevelee-civica  This should the be current version of the contact details which is on the ContactData property

File: src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml [hunk 2]
```diff
@@ -18,12 +18,68 @@
...
+                        <td class="govuk-table__cell">@Model.SmallProducerSubmissionData.CurrentSubmission.ContactAddressData.Email</td>
+                    </tr>
+
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">Phone number</th>
+                        <td class="govuk-table__cell">@Model.SmallProducerSubmissionData.CurrentSubmission.ContactAddressData.Telephone</td>
+                    </tr>
                 </tbody>
             </table>
+            <h1 class="govuk-heading-m">Contact address</h1>
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml:49]: @stevelee-civica  think for heading hierarchy this should probably be a h2, maybe worth checking with Wave

File: src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml [hunk 3]
```diff
@@ -18,12 +18,68 @@
             <h1 class="govuk-heading-m">
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml:18]: @stevelee-civica  think we probably need a back link

File: src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml (hunk 2 above)
Comment (stevelee-civica) [reply src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml]: changed to h2

File: src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml [hunk 4]
```diff
@@ -15,15 +15,71 @@
...
+                    </tr>
+
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">Position</th>
+                        <td class="govuk-table__cell">@Model.SmallProducerSubmissionData.ContactData.Position</td>
+                    </tr>
+
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">Email</th>
+                        <td class="govuk-table__cell">@Model.SmallProducerSubmissionData.CurrentSubmission.ContactAddressData.Email</td>
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml:40]: @stevelee-civica  looking at this it looks like we also need to have the Contact address data returned from GetSmallProducerSubmissionHandler. So add an AddressData property to SmallProducerSubmissionData and map that in 
image

Using the mapper.Map<Address, AddressData>(directRegistrant.Address) mapper similar to how the contact data is done and use that to display these fields.

Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml:40]: @stevelee-civica  and update the unit tests :)

Comment (stevelee-civica) [reply src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml]: updated

File: src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml (hunk 3 above)
Comment (stevelee-civica) [reply src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml]: already in place

**Pull request merged**